resolver/resolver.go

```diff
@@ -9,27 +9,27 @@ import (
 
 // Resolver is the parent task which is used to relay messages between the client, server and other sub-tasks.
 type Resolver struct {
-	logger    logrus.FieldLogger
-	client    tau.Task
-	server    tau.Task
-	p2p       tau.Task
-	addresses []addr.Addr
+	logger         logrus.FieldLogger
+	client         tau.Task
+	server         tau.Task
+	p2p            tau.Task
+	bootstrapAddrs []addr.Addr
```

Comment:
I think these addresses should not be called bootstrapAddrs. As they're not just the bootstrap nodes, they're the nodes in the shard which would be responsible for the computation. For now we only have one shard and the addresses are hard-coded. In the future, we will have another component giving us these addresses.